### ISSUE TITLE
progress bar when sending/uploading message

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-ios#536]
(especially needed when attachments are added)

if it's a bigger file, it takes a while: there should be a progress bar. We've done something similar on Android. The first 30% of the progress bar are taken up by encryption (which we have no way to track progress of) and the last 70% is upload of the message to Gmail servers, which you should be able to track progress of.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-ios/pull/505#pullrequestreview-763195500_

I need to link the Android issue later where it's explained in detail.

**Comment by tomholub** [comment]
Actually, on Android, we place this in the Outbox queue which gets sent asynchronously, without user waiting for it. But it took quite some effort to get that right, and handling errors is not easy. So we'll wait on this.

**Comment by tomholub** [comment]
duplicate of #695
